plugins/build.gradle.kts

```diff
@@ -18,9 +18,12 @@ import org.apache.tools.ant.filters.ReplaceTokens
 
 subprojects {
   repositories {
-    maven("https://repo.spongepowered.org/maven/")
```

Comment:
Do we really need all these repo changes here?

Reply:
Accidental merge conflicts ig